Description: Consider by-hash files as static data
Author: Colin Watson <cjwatson@ubuntu.com>
Bug-Debian: https://bugs.debian.org/819852

Index: b/source/acfg_defaults.cc
===================================================================
--- a/source/acfg_defaults.cc
+++ b/source/acfg_defaults.cc
@@ -30,6 +30,7 @@
 		"|[a-f0-9]+-(susedata|updateinfo|primary|deltainfo).xml.gz" //opensuse, index data, hash in filename
 		"|fonts/(final/)?[a-z]+32.exe(\\?download.*)?" // msttcorefonts, fonts/final/comic32.exe /corefonts/comic32.exe plus SF's parameters
 		"|/dists/.*/installer-[^/]+/[0-9][^/]+/images/.*" // d-i stuff with revision
+		"|/dists/.*/by-hash/.*"
     "|/[[:alpha:]]{1,2}/[a-f0-9]{64}(-[a-f0-9]{64})?(\\.gz)?" // FreeBSD, after https://alioth.debian.org/tracker/?func=detail&atid=413111&aid=315254&group_id=100566
 ")$");
 
